playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -156,6 +160,7 @@ def get_pod_containers(pod: Pod) -> List[RelatedContainer]:
                 memoryRequest=requests.memory,
                 restarts=getattr(containerStatus, "restartCount", 0),
                 status=stateStr,
+                status_details=status_details,
```

Comment:
Can you please change to camel case to be consistent with the other class fields?

Reply:
done